### PR TITLE
Updating Identity Engine > Embedded Authentication content for Node.js SDK Social IdP use case

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Adding Social IdP (Facebook) use case to Embedded Authentication SDK ([Sign in with Facebook](https://developer.okta.com/docs/guides/oie-embedded-sdk-use-cases/aspnet/oie-embedded-sdk-use-case-sign-in-soc-idp/))
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-408181](https://oktainc.atlassian.net/browse/OKTA-408181)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -1 +1,87 @@
...
+  await authClient.idx.handleInteractionCodeRedirect(callbackUrl);
+} catch (err) {
+  if (authClient.isInteractionRequiredError(err)) {
+    // need to proceed with required authenticator/s
+  }
+  throw err;
+}
+```
+
+With the obtained access token, you can retrieve basic user information by making a request to Okta's Open ID Connect authorization server. See [Get user profile information after sign in](/docs/guides/oie-embedded-sdk-alternate-flows/node.js/main/#getuserprofileinfo) for details.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/nodejs/integrationsteps.md:85]:
```suggestion
With the obtained access token, you can retrieve basic user information by making a request to Okta's OpenID Connect authorization server. See [Get user profile information after sign in](/docs/guides/oie-embedded-sdk-alternate-flows/node.js/main/#getuserprofileinfo) for details.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/nodejs/summaryofsteps.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+The following diagram shows the sequence of steps for the Facebook sign-in flow.
+
+<div class="common-image-format">
+
+![Displays the sequence diagram for Identity Engine embedded auth with Auth JS SDK - Sign in with Facebook use case](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-social-sign-in-nodejs.png)
+
+</div>
+
+>**NOTE:** The SDK sample application does not support this use case.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/nodejs/summaryofsteps.md:11]:
```suggestion
>**Note:** The SDK sample application doesn't support this use case.
```

**Review by susanharper-okta — APPROVED** [review]
LGTM - just a couple minor minor nits. Approving so I don't hold you up.

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/nodejs/summaryofsteps.md (hunk 2 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-soc-idp/nodejs/summaryofsteps.md:11]: Removed this note. Thanks.

**Pull request merged**